dashbuilder-website-docs/src/main/asciidoc/guides/yaml_guide.adoc

```diff
@@ -406,10 +406,176 @@ pages:
 .Dataset Lookup with group
 image::guides/yaml/groupLookup.png[Lookup with group]
 
-The supported group functions are `SUM`, `MAX`, `MIN` and `AVERAGE` for numbers. For label columns the supported functions are `DISTINCT` and `COUNT`.  
+In the example below the dataset lookup gives the median of the number of products per section:
```

Comment:
I think we need only one example for the group function and keep the explanation but include all others functions in the explanation